### ISSUE TITLE
Custom method for saving and restoring QAT items

### ISSUE THREAD

**Issue by floele-sp** [fluentribbon/Fluent.Ribbon#276]
Our application needs to store QAT items within an XML file. Currently the ribbon does not provide access to these items, however. My minimally invasive method to achieve my goal so far was changing 

// Currently added in QAT items
        private readonly Dictionary<UIElement, UIElement> quickAccessElements = new Dictionary<UIElement, UIElement>();

to 

// Currently added in QAT items
        protected readonly Dictionary<UIElement, UIElement> quickAccessElements = new Dictionary<UIElement, UIElement>();

and then implement the required functionality in a derived ribbon class. An officially supported way to do this would be great.

**Comment by batzen** [comment]
@floele-sp Could you have a look if my implementation fits your needs?

**Comment by floele-sp** [comment]
@batzen Thanks, the GetQuickAccessElements()-method should be sufficient.

**Comment by batzen** [comment]
@floele-sp So the implementation fits your needs, or not? ;-)

**Comment by floele-sp** [comment]
@batzen I cannot currently upgrade the Fluent library so I didn't use it yet, but in theory it fits my needs,

**Comment by batzen** [comment]
@floele-sp  Ok, then please post back as soon as you got the chance to verify my implementation.

**Comment by batzen** [comment]
@floele-sp Any news?

**Comment by floele-sp** [comment]
@batzen Sorry, but no. We'll need a few more weeks before we can upgrade, but I'll let you know. I probably need to report a few issues I noticed (if still present) anyway.

**Comment by batzen** [comment]
I don't like to pressure you, but this is the last open issue that's preventing version 5.0 from being released ;-)
Would be nice if you could have a look or just tell me to close this and if you encounter any problems open a new issue.

**Comment by batzen** [comment]
It's not the last open issue anymore. Moved some issues from 6.0 to 5.0. ;-)

**Comment by floele-sp** [comment]
I don't really want to stop you releasing 5.0, so feel free to close it.
I would certainly love trying the new version, but since we still haven't moved to .NET 4.5 (quite sad) I cannot compile the current version of Fluent for a test run.

**Comment by batzen** [comment]
Fluent still works with .net 4.0. You just need VS 2015 to compile. Have a look at the appveyor build, you can grab the compiled binaries from there. Will release the first official preview for v5 to nuget someday this week.

**Comment by floele-sp** [comment]
Oh I should probably have used the 4.0.csproj ;)

Alright, looks like the new property is working fine, thanks.

**Comment by batzen** [comment]
Thanks for testing.
